### ISSUE TITLE
HTTP Error 500: INTERNAL SERVER ERROR

### ISSUE THREAD

**Issue by B5r1oJ0A9G** [william-gr/freenas-bacula#1]
Not sure if it's a bug or my fault, because I'm not very familiar with the new "Plugins Jail" feature.

I think I've successfully installed the plugins jail and configured it correctly. The installation of the bacula-sd plugin ends successfully as well, but throws the following error message to the console:

```
Aug 11 10:00:46 freenas manage.py: [freeadmin.navtree:413] Couldn't retrieve https://192.0.43.10/plugins/bacula-sd/_s/treemenu: HTTP Error 500: INTERNAL SERVER ERROR
```

Trying to enable the plugin fails with the message 'Some error occured'.
Accessing the mentioned URL through a web browser let the assumption come up that there is an issue with the plugin's database:

```
DatabaseError at /plugins/bacula-sd/_s/treemenu

no such table: freenas_baculasdstorage

Request Method:     GET
Request URL:    https://192.0.43.10/plugins/bacula-sd/_s/treemenu
Django Version:     1.4
Exception Type:     DatabaseError
Exception Value:    

no such table: freenas_baculasdstorage

Exception Location:     /usr/pbi/bacula-sd-amd64/lib/python2.7/site-packages/django/db/backends/sqlite3/base.py in execute, line 337
Python Executable:  /usr/pbi/bacula-sd-amd64/bin/python
Python Version:     2.7.3
Python Path:    

['/usr/pbi/bacula-sd-amd64/baculaUI/../lib/python2.7/site-packages',
 '/usr/pbi/bacula-sd-amd64/lib/python2.7/site-packages',
 '/usr/pbi/bacula-sd-amd64',
 '/usr/pbi/bacula-sd-amd64/lib/python2.7/site-packages/setuptools-0.6c11-py2.7.egg',
 '/usr/pbi/bacula-sd-amd64/lib/python2.7/site-packages/South-0.7.5-py2.7.egg',
 '/usr/pbi/bacula-sd-amd64/lib/python2.7/site-packages/flup-1.0.2-py2.7.egg',
 '/usr/pbi/bacula-sd-amd64/lib/python2.7/site-packages/httplib2-0.7.4-py2.7.egg',
 '/usr/pbi/bacula-sd-amd64/lib/python2.7/site-packages/oauth2-1.5.211-py2.7.egg',
 '/usr/pbi/bacula-sd-amd64/lib/python27.zip',
 '/usr/pbi/bacula-sd-amd64/lib/python2.7',
 '/usr/pbi/bacula-sd-amd64/lib/python2.7/plat-freebsd10',
 '/usr/pbi/bacula-sd-amd64/lib/python2.7/lib-tk',
 '/usr/pbi/bacula-sd-amd64/lib/python2.7/lib-old',
 '/usr/pbi/bacula-sd-amd64/lib/python2.7/lib-dynload',
 '/usr/pbi/bacula-sd-amd64/lib/python2.7/site-packages']

Server time:    Sat, 11 Aug 2012 07:54:56 +0000
```

**Comment by B5r1oJ0A9G** [comment]
After issuing:

```
/var/db/pbi/installed/bacula-sd-5.2.6-amd64/post-install.sh
```

manually from within the plugins jail I was able to see the navigation tree in de UI.

**Comment by william-gr** [comment]
This should not be an issue in 8.3.0-BETA

**Comment by william-gr** [comment]
f06cf28e5783d6db8289c384a409e1fc9ca131af should fix the issue for 8.2.0

thanks
